### DISCUSSION TITLE
Roadmap

### DISCUSSION THREAD

**Discussion by candidosales** [chakra-ui/zag#94] [Q&A] [answered]
Hi everyone 👋,

Do you have any roadmap documents? I would like to see and maybe do some contribution :)

**Comment by timeTravelCYN** [comment]
@segunadebayo Hi, I would like to see and maybe do some contribution too!!!

**Comment by anubra266** [comment]
@candidosales @timeTravelCYN There's no public roadmap yet, but I think it'll be nice, if you just drop suggestions, as to what you think would be great additions to zagjs 🙂

**Comment by lightbluepoppy** [comment]
Hello everyone ٩( ᐛ )و

I am very curious about this project and hope to start using for my own design systems. This project let me create specific design systems for each of my projects very conveniently. So I need a roadmap to follow when this project reaches a stable release.

**Comment by segunadebayo** [comment, marked as answer]
You can find the roadmap here: https://ark-ui.canny.io/

Ark and Zag.js are tandem projects that share the same roadmap
